script/upgrades/MU08/MU08.md

```diff
@@ -0,0 +1,34 @@
+### MU08
+
+Transfer ownership of the MentoProtocol contracts to Mento Governance.
+
+Proposal Summary:
+This proposal transfers ownership of the MentoProtocol contracts to Mento Governance. This will allow Mento token holders to own and manage the protocol.
+
+Contracts to Transfer Ownership:
```

Comment:
Should we leave GovernanceFactory with Celo Gov? It can not do anything anymore and it is completely fine to not transfer it but I guess it would look weird if we keep using it as a reference.

Reply:
No I think it makes sense to take it with us 👍 
Good catch!